### PR TITLE
Add es_MX localization

### PR DESCRIPTION
**Pull request by DevonOrxi**

### CODE REVIEW

File: src/assets/locales/es_MX.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
+{
+  "originalReading": "Español de México",
+  "englishReading": "Spanish (Mexico)",
+  "actions": {
+    "add": "Agregar",
+    "added": "Agregado a la lista",
+    "addToPlanToWatch": "Add to plan to watch",
```
Comment (NicoAiko) [inline src/assets/locales/es_MX.json:7]: You missed this.

File: src/assets/locales/es_MX.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+      },
+      "changelog": {
+        "new": "NUEVO",
+        "fix": "CORREGIDO",
+        "remove": "REMOVIDO",
+        "version": "Versión",
+        "changesIn": "Últimos cambios en {0}"
+      },
+      "specialThanks": "Agradecimientos especiales",
+      "supporters": "Supporters"
```
Comment (NicoAiko) [inline src/assets/locales/es_MX.json:191]: Supporters is `Supporters` in Spanish too?

File: src/assets/locales/es_MX.json (hunk 1 above)
Comment (DevonOrxi) [reply src/assets/locales/es_MX.json:7]: Thanks, missed it indeed. On it.

File: src/assets/locales/es_MX.json (hunk 2 above)
Comment (DevonOrxi) [reply src/assets/locales/es_MX.json:191]: Oh, forgot to tackle this. It's a really difficult term to translate to Spanish. Maybe we can use `Supported by` in Spanish for this one?

Comment (NicoAiko) [reply src/assets/locales/es_MX.json]: Sure. The string is a headline for a list of supporters, so supported by is a valid change.
Thanks!

**Pull request merged**